pokemongo_bot/tests/bot_test.py

```diff
@@ -0,0 +1,574 @@
+                           event_manager, mapper, stepper, navigator)
+
+        assert len(bot.pokemon_list) == 151
+        assert len(bot.item_list) == 30
+        assert bot.position == (0, 0, 0)
+
+        assert bot.config is config_namespace
+        assert bot.api_wrapper is api_wrapper
+        assert bot.player_service is player_service
+        assert bot.pokemon_service is pokemon_service
+        assert bot.plugin_manager is plugin_manager
+        assert bot.event_manager is event_manager
+        assert bot.mapper is mapper
+        assert bot.stepper is stepper
+        assert bot.navigator is navigator
+
+    @patch('pokemongo_bot.event_manager.manager', new_callable=EventManager)
+    @patch('pokemongo_bot.logger.log', return_value=None)
+    def test_start_login_success_no_debug(self, log_fn, event_manager):
+        bot = self._create_generic_bot({
+            'debug': False,
+            'test': False,
+            "print_events": True,
+            'exclude_plugins': ['another_test_plugin'],
+            'load_library': 'libencrypt.so',
+            'auth_service': 'ptc',
+            'username': 'test_account',
+            'password': 'pa55w0rd',
+            'location': '51.5037053,-0.2047603',
+            'location_cache': False,
+            'gmapkey': 'test_gmaps_key',
+            'initial_transfer': True,
+            'recycle_items': True
+        })
+
+        bot.mapper.google_maps.elevation = Mock(return_value=[{'elevation': 10.1}])
+
+        event_manager.fire_with_context = Mock()
+        bot.event_manager = event_manager
+
+        pgo = bot.api_wrapper.get_api()
+        pgo.set_response('get_player', self._create_generic_player_response())
+        pgo.set_response('get_inventory', self._create_generic_inventory_response())
+
+        bot.start()
+
+        assert log_fn.call_count == 24
+        log_fn.assert_any_call('Plugins loaded: [\'test_plugin\']', color='green', prefix='Plugins')
+        log_fn.assert_any_call('Events available: [\'test\']', color='green', prefix='Events')
+
+        log_fn.assert_any_call('Username: test_account', prefix='#')
+        log_fn.assert_any_call('Bag storage: 36/350', prefix='#')
+        log_fn.assert_any_call('Pokemon storage: 2/250', prefix='#')
+        log_fn.assert_any_call('Stardust: 20,000', prefix='#')
+        log_fn.assert_any_call('Pokecoins: 10', prefix='#')
+        log_fn.assert_any_call('Poke Balls: 11', prefix='#')
+        log_fn.assert_any_call('Great Balls: 12', prefix='#')
+        log_fn.assert_any_call('Ultra Balls: 13', prefix='#')
+        log_fn.assert_any_call('-- Level: 14', prefix='#')
+        log_fn.assert_any_call('-- Experience: 15', prefix='#')
+        log_fn.assert_any_call('-- Experience until next level: 985', prefix='#')
+        log_fn.assert_any_call('-- Pokemon captured: 17', prefix='#')
+        log_fn.assert_any_call('-- Pokestops visited: 18', prefix='#')
```

Comment:
This logging is now handled by the player service, so i'm going to remove these assertions in favour of asserting that the player service `print_stats` method is called.